src/renderer/src/routes/modals/SellTokensDeposit/SellTokensDepositModal.tsx

```diff
@@ -0,0 +1,465 @@
+  Token,
+} from '@cityofzion/blockchain-service'
+import { ActionStep } from '@renderer/components/ActionStep'
+import { ActionStepSeparator } from '@renderer/components/ActionStepSeparator'
+import { AlertErrorBanner } from '@renderer/components/AlertErrorBanner'
+import { Button } from '@renderer/components/Button'
+import { GreyAccountSelect } from '@renderer/components/GreyAccountSelect'
+import { GreyAmountInput } from '@renderer/components/GreyAmountInput'
+import { GreyTokenSelect } from '@renderer/components/GreyTokenSelect'
+import { Input } from '@renderer/components/Input'
+import { Separator } from '@renderer/components/Separator'
+import { TransactionFeeActionStep } from '@renderer/components/TransactionFeeActionStep'
+import { AccountHelper } from '@renderer/helpers/AccountHelper'
+import { NumberHelper } from '@renderer/helpers/NumberHelper'
+import { ToastHelper } from '@renderer/helpers/ToastHelper'
+import { UtilsHelper } from '@renderer/helpers/UtilsHelper'
+import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
+import { useActions } from '@renderer/hooks/useActions'
+import { useCurrentLoginSessionSelector } from '@renderer/hooks/useAuthSelector'
+import { useBalance } from '@renderer/hooks/useBalances'
+import { useDepositActionsDataSelector } from '@renderer/hooks/useBuyAndSellTokensSelector'
+import { useModalNavigate, useModalState } from '@renderer/hooks/useModalRouter'
+import { useNameService } from '@renderer/hooks/useNameService'
+import { useAppDispatch } from '@renderer/hooks/useRedux'
+import { useCurrencySelector, useSelectedNetworkByBlockchainSelector } from '@renderer/hooks/useSettingsSelector'
+import { SideModalLayout } from '@renderer/layouts/SideModal'
+import { bsAggregator } from '@renderer/libs/blockchainService'
+import { authReducerActions } from '@renderer/store/reducers/AuthReducer'
+import { buyAndSellTokensReducerActions } from '@renderer/store/reducers/BuyAndSellTokensReducer'
+import { TBlockchainServiceKey } from '@shared/@types/blockchain'
+import { TDepositActionsData } from '@shared/@types/buy-and-sell-tokens'
+import { TUseTransactionsTransfer } from '@shared/@types/hooks'
+import { IAccountState } from '@shared/@types/store'
+
+import { SellTokensDepositErrorContent } from './SellTokensDepositErrorContent'
+import { SellTokensDepositSuccessContent } from './SellTokensDepositSuccessContent'
+
+type TLocationState = {
+  account?: IAccountState
+}
+
+export const SellTokensDepositModal = () => {
+  const { t } = useTranslation('modals', { keyPrefix: 'sellTokensDeposit' })
+  const dispatch = useAppDispatch()
+  const { currentLoginSessionRef } = useCurrentLoginSessionSelector()
+  const { accountsRef } = useAccountsSelector()
+  const { modalNavigate } = useModalNavigate()
+  const { networkByBlockchain } = useSelectedNetworkByBlockchainSelector()
+  const { depositActionsData } = useDepositActionsDataSelector()
+  const { currencyRef } = useCurrencySelector()
```

Comment:
use `currency` instead currencyRef

Reply:
Why? The currency won't change on this screen.